### DISCUSSION TITLE
Full tutorial?

### DISCUSSION THREAD

**Discussion by EraseRewin** [SunOner/yolov8_aimbot#38] [General]

**Comment by SunOner** [comment]
Yes. Install cudnn.

**Reply by SunOner** [reply]
The model was not exported to the end, an export error occurred. Delete the onnx model before exporting. Pip uninstall onnx, then export the model.

**Reply by SunOner** [reply]
try to set fps to 30 and mouse smooth to 1

**Comment by EraseRewin** [comment]


**Reply by SunOner** [reply]
no need to perform the permutation completely, just download the new yolov8_aimbot
